chapters/04_game_theoretic_model/03_methodology/main.tex

```diff
@@ -0,0 +1,1327 @@
+
+The first case of equation~(\ref{eq:obj_distributor_no_brent_method})
+corresponds to the event where both \(f(0)\) and \(f(1)\) are negative.
+Therefore, for all values of \(p_A \in [0, 1]\) the objective function is
+negative, which means that:
+\[
+    \alpha L_A(p_A) + (1 - \alpha) B_A(p_A) <
+    \alpha L_B(p_B) - (1 - \alpha) B_B(p_B), \quad \text{for all } p_A \in [0,1]
+\]
+
+Thus, the distribution service's best response would be to send all individuals
+to queueing system A (\(p_A=1, p_B=0\)).
+Similarly, the second case of
+equation~(\ref{eq:obj_distributor_no_brent_method}) corresponds to the event
+where for all values of \(p_A \in [0, 1]\) the objective function is
+positive, which means that:
+\[
+    \alpha L_A(p_A) + (1 - \alpha) B_A(p_A) >
+    \alpha L_B(p_B) - (1 - \alpha) B_B(p_B), \quad \text{for all } p_A \in [0,1]
+\]
+Equivalently, this indicates that the distribution service's best response
+would be to send all individuals to queueing system B (\(p_A=0, p_B=1\)).
+Therefore, the methodology that is used to find the best \(p_A\) that satisfies
+equation~(\ref{eq:obj_distributor_2}) can by calculated in the following way:
+
+\begin{equation}\label{eq:obj_distributor_implementation}
+    p_A = \begin{cases}
+        1, & \text{if } O(0) < 0 \text{ and } O(1) < 0 \\
+        0, & \text{if } O(0) > 0 \text{ and } O(1) > 0 \\
+        \text{Use Brent's method}, & \text{if } O(0)O(1) < 0
+    \end{cases}
+\end{equation}
+
+where \(O(p_A)\) is the objective function of the distribution service described
+in equation~(\ref{eq:obj_distributor_2}).
+
+\subsubsection{Examples}\label{sec:brent_method_example}
+Consider a distribution service whose arrival rate of type 2 individuals is
+\(\lambda_2 = 4\) and the `importance'-parameter is \(\alpha = 0.2\).
+Additionally, let queueing system \(A\) and queueing system \(B\) have the
+following parameters:
+
+\begin{multicols}{2}
+    \begin{itemize}
+        \item \(\lambda_1^A = 2\)
+        \item \(\mu^A = 3\)
+        \item \(C^A = 3\)
+        \item \(T^A = 8\)
+        \item \(N^A = 15\)
+        \item \(M^A = 10\)
+        \item \(\lambda_1^B = 1\)
+        \item \(\mu^B = 1\)
+        \item \(C^B = 3\)
+        \item \(T^B = 10\)
+        \item \(N^B = 10\)
+        \item \(M^B = 5\)
+    \end{itemize}
+\end{multicols}
+
+The distribution service's best response for this particular example can be
+found at the intersection of the two decision values of the two queueing
+systems over \(p_A\).
+Figure~\ref{fig:brent_method_example} illustrates the distribution service's
+best response for this particular example.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\textwidth]{chapters/04_game_theoretic_model/img/brents_method/brent_method_example.pdf}
+    \caption{Decision values for queueing system A and queueing system B}
+    \label{fig:brent_method_example}
+\end{figure}
+
+In order to apply Brent's method to the current example the differences between
```

Comment:
```suggestion
In order to apply Brent's alogithm to the current example the differences between
```

You seem to interchange `Brent` and `brent` as well as `algorithm` and `method`. It should be `Brent`, and I think I prefer `alogorithm`.

Reply:
Changed it everywhere to be Brent's algorithm

Reply:
Changed it everywhere to Brent's algorithm for now